### PR TITLE
Removed protobuf-to-swagger conversion artifacts

### PR DESCRIPTION
**Pull request by delagoya**
* removed any `title: OPTIONAL` annotations that did not make sense
* added `required: []` for required parameters
* Added schema version, updating minor version since prefix may be a API breaking change

Signed-off-by: Angel Pizarro <delagoya@gmail.com>

### CODE REVIEW

**Comment by susheel** [comment]
@delagoya Shall I point this PR to `develop` and review changes?

**Comment by delagoya** [comment]
Yep

File: openapi/task_execution.swagger.yaml [hunk 1]
```diff
@@ -547,14 +526,17 @@ definitions:
...
         type: string
         description: |-
           Date + time the task was created, in RFC 3339 format.
           This is set by the system, not the client.
-        title: 'OUTPUT ONLY, REQUIRED'
+        readOnly: true
     description: Task describes an instance of a task.
+    required:
+    - executors
+    - creation_time
```
Comment (adamstruck) [inline openapi/task_execution.swagger.yaml]: OpenAPI 2.0 says parameters marked as `readOnly` should not be in the required list of the schema. So you should remove `creation_time` here.

Comment (susheel) [reply openapi/task_execution.swagger.yaml]: Commit 8f483cec91076eb14fb39e638d6d71df8cf9953d removed `creation_time` from the list of required params for `tesTask`

**Comment by susheel** [comment]
@delagoya @adamstruck Are you happy to merge?

**Pull request merged**